content/en/docs/setup/install/gateway/index.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Installing Gateways
+description: Install and customize Istio Gateways.
+weight: 40
+keywords: [install,gateway,kubernetes]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+Along with creating a service mesh, Istio allows managing [gateways](/docs/concepts/traffic-management/#gateways),
+which run at the edge of the mesh and allow fine-grained control over traffic entering and leaving the mesh.
+
+Follow this guide to deploy one or more gateways.
+
+## Prerequisites
+
+This guide requires the Istio control plane [to be installed](/docs/setup/install/) before proceeding.
+
+## Installing the gateway with injection
+
+To support users that already utilize existing deployment tools, we provide a few different ways to deploy a gateway.
+Each method will result in the same outcome; we recommend choosing the method you are most familiar with.
+If you are not sure which to choose, we suggest the Simple YAML method.
```

Comment:
Why are we recommending that one? It actually looks the most complicated in terms of what the user needs to manage.

Reply:
Its subjective which is "best", but I prefer it because:
* Its using plain Kubernetes APIs, which are understood by all users, rather than our own (undocumented) Helm values API
* It is the most flexible
* It is the only way we have right now to safely canary upgrade a gateway. The (current unwritten) upgrade section will likely depend on this for some upgrade options
* No extra tools required

I would not be surprised at all if many disagree. But if I was not familiar with helm, I would probably not want to go learn a new tool; If I am familiar with helm the line above recommends choosing what the user is familiar with.

I am ok with changing the recommended path if others have a different preference

Reply:
Well if they're using Istio they're already likely using either istioctl or helm right? It seems unlikely they don't have either of those...And both of those are MUCH shorter than the full yaml.

To avoid this we could just drop the sentence and not make a recommendation at all, then the previous sentence just tells them to use what they're familiar with and we're good.